client/packages/common/src/intl/number/IntlNumber.ts

```diff
@@ -15,7 +15,11 @@ export const useFormatNumber = () => {
     ) => {
       if (value === undefined) return '';
       const locale = options?.locale ?? currentLanguage;
-      return new Intl.NumberFormat(locale, options).format(value);
+      return new Intl.NumberFormat(locale, {
+        // Don't round here, as most use cases have their own internal rounding
```

Comment:
I don't believe the comment is accurate!
Have looked for all usages of `format` and only the NumericTextInput is rounding, though not with options to the format (`setTextValue(format(constrained))`) 

The usages are in:
* NumericTextInput (this relies on downstream usage specifying dp which I think they all do)
* StockItemSearchInput
* StockItemSearchInputWithStats
* OutboundLineEditForm
* PrescriptionLineEditForm

all the others are using integers, so won't have an issue with the change.

Maybe just remove the comment. it only really makes sense in light of the change

Reply:
Yeah, it made more sense when I was considering a few additional updates.